### PR TITLE
chore(TUP-30992)CVE: junit-4.11,4.13

### PR DESCRIPTION
**Pull request by jiezhang-tlnd**

### CODE REVIEW

File: junit/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Junit
+Bundle-SymbolicName: org.junit
+Bundle-Version: 4.13.2
+Automatic-Module-Name: org.junit
+Bundle-RequiredExecutionEnvironment: JavaSE-5
+Require-Bundle: org.hamcrest.core;bundle-version="1.3.0";visibility:=r
+ eexport`
```
Comment (bhe-talendbj) [inline junit/META-INF/MANIFEST.MF:9]: why has a "`"?

File: junit/.classpath [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry exported="true" kind="lib" path="lib/junit-4.13.2.jar"/>
```
Comment (bhe-talendbj) [inline junit/.classpath:3]: remove this line

**Review by bhe-talendbj — CHANGES_REQUESTED** [review]

File: junit/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -0,0 +1,41 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Junit
+Bundle-SymbolicName: org.junit
+Bundle-Version: 4.13.2
+Automatic-Module-Name: org.junit
+Bundle-RequiredExecutionEnvironment: JavaSE-5
```
Comment (bhe-talendbj) [inline junit/META-INF/MANIFEST.MF:7]: remove this? can you please help compare this with existing manifest.mf of 4.13.0?

File: junit/pom.xml [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+                        <goals>
+                            <goal>bundle</goal>
+                        </goals>
+                        <configuration>
+                            <classifier>bundle</classifier>
+                            <instructions>
+                                <Bundle-SymbolicName>org.junit</Bundle-SymbolicName>
+                                <Service-Component>*</Service-Component>
+                                <DynamicImport-Package>*</DynamicImport-Package>
+                                <Require-Capability>osgi.ee;filter:="(&amp;(osgi.ee=JavaSE)(version=1.5))"</Require-Capability>
```
Comment (bhe-talendbj) [inline junit/pom.xml:88]: version=1.8

File: junit/META-INF/MANIFEST.MF (hunk 1 above)
Comment (jiezhang-tlnd) [reply junit/META-INF/MANIFEST.MF]: removed. thx

**Review by bhe-talendbj — APPROVED** [review]

**Pull request merged**